_posts/2019-10-17-how-to-achieve-cis-benchmark-compliance.adoc

```diff
@@ -1466,36 +1472,24 @@ Set the variables for the `account-baseline-app` module in this environment in t
 [source,hcl]
 ----
 locals {
-  aws_region = "us-east-1"
-
-  accounts = {
-    logs     = "409800740445"
-    security = "673123100581"
-    shared   = "384759303421"
-    dev      = "293847503945"
-    stage    = "384924092834"
-    prod     = "784260063686"
-    root     = "216044045972"
-  }
+  # A local for more convenient access to the accounts map.
+  accounts = local.common_vars.locals.accounts
```

Comment:
Where is `common_vars` defined?

Reply:
In the root `terragrunt.hcl` file.

Reply:
This is so we don't have multiple places defining the same thing. If it's easier though I can copy paste the stuff from LZ guide into here?

Reply:
Do `local` vars inherit from root `terragrunt.hcl` files? I didn't think they did...

Reply:
Should this be like below? 
```
locals {
  common_vars = read_terragrunt_config(find_in_parent_folders("common.hcl"))
}

inputs = merge(
  local.common_vars.inputs,
  {
    # additional inputs
  }
)
```

Reply:
Yea, that's what I would expect